homeassistant/data_entry_flow.py

```diff
@@ -207,6 +207,13 @@ async def async_create_flow(
         Handler key is the domain of the component that we want to set up.
         """
 
+    async def async_flow_aborted(
+        self,
+        flow: FlowHandler[_FlowContextT, _FlowResultT, _HandlerT],
+        result: _FlowResultT,
+    ) -> None:
+        """Handle an aborted data entry flow."""
```

Comment:
If we go for this change, the docstring needs to be improved to make it clear exactly when the method is called. Also, we should maybe change the name of the new method.

Reply:
No longer relevant